dataminer-overview/Support Services/Support_services_agreement.md

```diff
@@ -69,13 +69,13 @@ The User shall ensure that they have an industry-standard and uninterrupted remo
 
 ## Ticket generation
 
-When the User encounters an Issue with the Software or wishes to receive Support for another reason, the User can at their choice [open a ticket for Support or call Skyline's Support helpdesk](https://skyline.be/contact/tech-support).
+When the User encounters an Issue with the Software or wishes to receive Support for another reason, the User should open a ticket for Support using the [User Operations Support web application](https://operations.skyline.be/Home/CreateTicket). More details about this can be found in the [User Operations Support documentation](xref:User_operations_support).
 
-Tickets for Support can be generated through dataminer.services.
+If for some reason the User cannot access the User Operations Support web application and create a ticket, they can also [contact Skyline's Support helpdesk](https://skyline.be/contact/tech-support).
 
 Support is available during Business Hours only, except if the User has purchased an Out of Business Hours Support plan. If the User wishes to create a ticket in follow-up of their telephone support, the User needs to generate a ticket through Skyline's dataminer.services.
```

Comment:
@XeniaHenning, I've made some changes based on your feedback, but I'm still hesitating whether we should keep this sentence:  "If the User wishes to create a ticket in follow-up of their telephone support, the User needs to generate a ticket through Skyline's dataminer.services."
Possibly we can keep it, but the ticket must be made with the new app instead?

Reply:
but the ticket must be made with the new app instead > totally agree.

Reply:
I've adjusted the sentence!